test/inductor/test_cpu_repro.py

```diff
@@ -2190,6 +2142,7 @@ def fn(a, b, c, idx):
             self.assertEqual(metrics.generated_kernel_count, 1)
             self.assertTrue(same(fn(a, b, c, idx), opt_fn(a, b, c, idx)))
 
+    @requires_vectorization
     def test_lowp_fp_neg_abs(self):
```

Comment:
Can we guard it around `metrics.generated_cpp_vec_kernel_count` checks?

Reply:
Yes, but it may not make sense to run that test then. I took a look where such a guard was already used and only common thing I found is a `self.common(fn1, (x,))` call inside, which isn't here. See the PR description.

I assume w/o vectorization `opt_fn == fn` is trivially true and hence not worth checking. That could be the reason why some tests where skipped before vs. guarded checks

Reply:
> I assume w/o vectorization `opt_fn == fn` is trivially true and hence not worth checking. That could be the reason why some tests where skipped before vs. guarded checks

Hmm, why `opt_fn == fn` w/o vectorization? `opt_fn` is from `torch.compile` and runs with cpp codegen while `fn` is in eager mode.

Reply:
> Hmm, why `opt_fn == fn` w/o vectorization

I was wondering if that compilation really always happens without vectorization. But as mentioned this is only guessing on why some parts use a skip-macro and other an `if`-check.

Reply:
Skip marker removed after guarding the check